### PR TITLE
adds "Butler" Assistant alternate title

### PR DESCRIPTION
**Pull request by Boviro**
Adds Butler alt title for assistant

## About The Pull Request

Adds 1 line, 9 characters to alt_job_titles.dm, adding Butler as an assistant alt title

## Why It's Good For The Game

Adds some refined butlerly spirit to our beloved, but unrefined, Bubberstation.

## Proof Of Testing

Tested: Compiles and runs without errors and functions properly in-game
image
image
image
image

## Changelog

add: new Butler alt title for assistant

:cl:
add: Added butler as assistant title
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm [hunk 1]
```diff
@@ -22,6 +22,7 @@
 		"Artist",
 		"Businessman",
 		"Businesswoman",
+		"Butler",
```
Comment (projectkepler-ru) [inline modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm:25]:
```suggestion
		"Butler", //Bubber Addition
```

**Comment by projectkepler-ru** [comment]
Oh for fuck sake

**Review by projectkepler-ru — APPROVED** [review]
Honestly, you could just do this on upstream skyrat but its fine otherwise

**Review by Majkl-J — CHANGES_REQUESTED** [review]
Make modular with
```
/datum/job/assistant/New()
	alt_titles += list("Butler")
```

**Comment by Boviro** [comment]
so adding alt titles in the modular_zubbers alt_job_titles.dm , as opposed to the modular_skyrat alt job titles module, correct?

**Comment by Majkl-J** [comment]
> so adding alt titles in the modular_zubbers alt_job_titles.dm , as opposed to the modular_skyrat alt job titles module, correct?

Yup!

**Comment by Boviro** [comment]
Hold please. Need to test locally first to ensure the change worked.

**Comment by Boviro** [comment]
Can confirm: Functions as intended

**Review by Majkl-J — APPROVED** [review]
Code looks solid

**Pull request merged**